### PR TITLE
Add new HTML editor icon for Insert Emoticon toolbar button

### PR DESCRIPTION
**Pull request by grant-cleary**
I think the title says it all in this case. :)

### CODE REVIEW

File: images/html-editor/insert-emoticon.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
```
Comment (dlockhart) [inline images/html-editor/insert-emoticon.svg]: Can you remove the `class="st0"` from the elements? Not needed.

Comment (grant-cleary) [reply images/html-editor/insert-emoticon.svg]: Sure, can do!

Comment (grant-cleary) [reply images/html-editor/insert-emoticon.svg]: Ah, whoops, those also show in the HTML file. I'll also remove them there.

**Comment by grant-cleary** [comment]
Forgot to run the build to update the SCSS file. Will merge after CI passes if there are no objections. :)

**Pull request merged**